src/proto/single/ent.rs

```diff
@@ -0,0 +1,130 @@
+use chrono::{DateTime, Utc};
+
+use crate::JobBuilder;
+
+impl JobBuilder {
+    /// When Faktory should expire this job.
+    ///
+    /// Faktory Enterprise allows for expiring jobs. This is setter for `expires_at`
+    /// field in the job's custom data.
+    /// ```
+    /// use faktory::JobBuilder;
+    /// use chrono::{Duration, Utc};
+    ///
+    /// let _job = JobBuilder::new("order")
+    ///     .args(vec!["ISBN-13:9781718501850"])
+    ///     .expires_at(Utc::now() + Duration::hours(1))
+    ///     .build();
+    /// ```
+    pub fn expires_at(&mut self, dt: DateTime<Utc>) -> &mut Self {
+        self.add_to_custom_data(
+            "expires_at".into(),
+            dt.to_rfc3339_opts(chrono::SecondsFormat::Nanos, true),
+        )
+    }
+
+    /// In what period of time from now (UTC) the Faktory should expire this job.
+    ///
+    /// Use this setter when you are unwilling to populate the `expires_at` field in custom
+    /// options with some exact date and time, e.g.:
```

Comment:
Given the implementation of this method is to compute `Utc::now` and then set an absolute time, this comment is a little misleading — a caller might reasonably read it and conclude that they can use this to use the server's time or something like that. May be worth clarifying that this just computes `now + ttl` under the hood.